tests/Dom/DocumentTest.php

```diff
@@ -1471,6 +1471,22 @@ public function dataSVGSelfClosingElements()
                 . '    </g>'
                 . '</svg>'
                 . '</body></html>',
+            ],
+            'test g tag one path child and newlines' => [
+                '<!DOCTYPE html><html><head></head><body>' . PHP_EOL
+                . '<svg>' . PHP_EOL
+                . '  <g id="ok">' . PHP_EOL
+                . '    <path/>' . PHP_EOL
+                . '  </g>' . PHP_EOL
+                . '</svg>' . PHP_EOL
+                . '</body></html>',
+                '<!DOCTYPE html><html>' . $head . '<body>' . PHP_EOL
+                . '<svg>' . PHP_EOL
+                . '  <g id="ok">' . PHP_EOL
+                . '    <path/>' . PHP_EOL
```

Comment:
This is currently failing in `assertSimilarMarkup`:

```
1) AmpProject\Dom\DocumentTest::testSVGSelfClosingElements with data set "test g tag one path child and newlines" ('<!DOCTYPE html><html><head></.../html>', '<!DOCTYPE html><html><head><m.../html>')
Failed asserting that two arrays are equal.
--- Expected
+++ Actual
@@ @@
     5 => '<body>'
     6 => '<svg>'
     7 => '<g id="ok">'
-    8 => '<path/>'
+    8 => '<path>'
     9 => '</g>'
     10 => '</svg>'
     11 => '</body>'
     12 => '</html>'
 )
```

Reply:
Addressed in cb091ed07 by using `assertEqualMarkup` instead of `assertSimilarMarkup`.